config/install/tripal.tripal_content_terms.trpcultivate_terms.yml

```diff
@@ -219,3 +228,15 @@ vocabularies:
             -   id: 'OBI:0001636'
                 name: 'grant'
                 description: 'A plan specification of organization A to give money to organization B so that B conducts investigations. Organization A has funder role and Organization B has research organization role.'
+    -   name: 'germplasm_ontology'
+        label: 'GCP germplasm ontology'
+        url: 'https://github.com/UofS-Pulse-Binfo/kp_entities/blob/master/ontologies/'
+        idSpaces:
+            -   name: 'CO_010'
+                description: 'Crop Germplasm Ontology'
+                urlPrefix: "https://github.com/UofS-Pulse-Binfo/kp_entities/blob/master/ontologies/{accession}.obo"
+        terms:
+            -   id: 'CO_010:0000159'
+                name: '400 breedingmaterial'
+            -   id: 'CO_010:0000348'
+                name: 'collaborator'
```

Comment:
As mentioned in other comments, we've decided not to use these terms. As such, we can remove this section entirely from this YAML.